src/defrag.c

```diff
@@ -1041,8 +1041,12 @@ Defrag(ThreadVars *tv, DecodeThreadVars *dtv, Packet *p)
 
     /* return a locked tracker or NULL */
     tracker = DefragGetTracker(tv, dtv, p);
-    if (tracker == NULL)
-        return NULL;
+    if (tv != NULL && dtv != NULL) {
```

Comment:
this is wrong. The error check should not depend on tv/dtv. Only the stats increase should depend on it